### PR TITLE
Rotate envmaps to be consistent with vanilla

### PR DESCRIPTION
**Pull request by gheskett**
The script has been modified to support CCW envmap rotation; the in-game DLs still need updated before this can be opened fully.

### CODE REVIEW

**Review by thecozies — CHANGES_REQUESTED** [review]

File: actors/mario/model.inc.c [hunk 1]
```diff
@@ -3250,7 +3250,7 @@ const Gfx mario_metal_medium_poly_butt[] = {
     gsSPSetGeometryMode(G_TEXTURE_GEN),
     gsDPSetCombineMode(G_CC_DECALFADE, G_CC_DECALFADE),
     gsDPLoadTextureBlock(mario_texture_metal, G_IM_FMT_RGBA, G_IM_SIZ_16b, 64, 32, 0, G_TX_WRAP | G_TX_NOMIRROR, G_TX_WRAP | G_TX_NOMIRROR, 6, 5, G_TX_NOLOD, G_TX_NOLOD),
-    gsSPTexture(0x0F80, 0x07C0, 0, G_TX_RENDERTILE, G_ON),
+    gsSPTexture(0x07C0, 0x0F80, 0, G_TX_RENDERTILE, G_ON),
```
[the remark this answers is not in the document]
Comment (thecozies) [reply actors/mario/model.inc.c:3253]: ill fix it

[the remark this answers is not in the document]
Comment (Mineqwerty) [reply actors/mario/model.inc.c:3253]: petervoid

**Review by thecozies — APPROVED** [review]

**Review by thecozies — CHANGES_REQUESTED** [review]
need:
```
inside_castle_seg7_texture_light_radial_inverse - levels/castle_inside/16.ia16.inc.c
cotmc_seg7_texture_07001800 - levels/cotmc/2.rgba16.inc.c

gd_texture_red_star_0
gd_texture_red_star_1
gd_texture_red_star_2
gd_texture_red_star_3
gd_texture_red_star_4
gd_texture_red_star_5
gd_texture_red_star_6
gd_texture_red_star_7
gd_texture_white_star_0
gd_texture_white_star_1
gd_texture_white_star_2
gd_texture_white_star_3
gd_texture_white_star_4
gd_texture_white_star_5
gd_texture_white_star_6
gd_texture_white_star_7
gd_texture_sparkle_0
gd_texture_sparkle_1
gd_texture_sparkle_2
gd_texture_sparkle_3
gd_texture_sparkle_4
gd_texture_mario_face_shine

paintings:
inside_castle_seg7_texture_07017000 ddd
inside_castle_seg7_texture_07016800 hmc
hmc_seg7_texture_07024CE0 levels/hmc/areas/1/painting.inc.c

```

**Comment by Mineqwerty** [comment]
the painting textures are actually already in here
image

**Review by gheskett — COMMENTED** [review]
Approved kinda sorta I think (I cannot review my own PRs) (also I am just kinda assuming everything I didn't do is good here)

**Pull request merged**